tests/test_cli/test_generate/test_protocols.py

```diff
@@ -45,7 +45,7 @@
 )
```

Comment:
Disabled due to consistently failing.

@DavidMinarsch sould be re-enabled before merging? or keep disabled until fixed?

Reply:
Reenable please;